lib/matplotlib/axes/_axes.py

```diff
@@ -1433,7 +1433,8 @@ def plot_date(self, x, y, fmt='o', tz=None, xdate=True, ydate=False,
 
     # @_preprocess_data() # let 'plot' do the unpacking..
     @docstring.dedent_interpd
-    def loglog(self, *args, **kwargs):
+    def loglog(self, *args, basex=10, basey=10, subsx=None, subsy=None,
+               nonposx='mask', nonposy='mask', **kwargs):
```

Comment:
I think it looks slightly better (though semantically identical) to have e.g.
```
def loglog(self, *args,
           basex=..., subsx=..., ...
           basey=..., ...
           **kwargs):
```
(but ymmv)